runtime/caml/domain_state.tbl

```diff
@@ -84,3 +84,6 @@ DOMAIN_STATE(uintnat, eventlog_paused)
 DOMAIN_STATE(uintnat, eventlog_enabled)
 DOMAIN_STATE(FILE*, eventlog_out)
 /* See eventlog.c */
+
+DOMAIN_STATE(void*, checking_pointer_pc)
```

Comment:
This field is only needed for the checker, so given that we'd want to remove it in the future, is it worth protecting it with `#ifdef` for the PR?

Reply:
Regardless, the field isn't used on Windows, so it'd be better not to have it (as it'll create a hard error if it's then accidentally referred to)